plugin/plugin.go

```diff
@@ -140,8 +210,10 @@ func Exec(ctx context.Context, args Args) (err error) {
 
 	if args.JsonFile != "" {
 		jsonData := JsonOutput{
-			Token: tokenData,
-			Jwt:   jwtSigned,
+			Token:           tokenData,
+			Jwt:             jwtSigned,
+			RepositoryCount: len(tokenData.Repositories),
+			Permissions:     tokenData.Permissions,
```

Comment:
```suggestion
			Token:           tokenData,
			Jwt:             jwtSigned,
```
please remove `RepositoryCount` and `Permissions` here